gprofiler/profilers/profiler_base.py

```diff
@@ -122,3 +138,28 @@ def snapshot(self) -> ProcessToStackSampleCounters:
                     logger.exception(f"{self.__class__.__name__}: failed to profile process {futures[future]}")
 
         return results
+
+
+def _handle_kernel_messages(messages):
```

Comment:
I don't think we should do it in `profiler_base`, since we'll get logs for processes of all types, while we currently care only about Java processes (since async-profiler is the only intrusive profiler)